doc/manuals/orion-api.md

```diff
@@ -1012,10 +1012,7 @@ and/or those which match a query or geographical query (see [Simple Query Langua
 (i.e. mutually exclusive) with their corresponding exact matching parameters, i.e. `idPattern` with `id` and
 `typePattern` with `type`.
 
-The response payload is an array containing one object per matching entity. Each entity follows
-the JSON entity representation format (described in "JSON Entity Representation" section).
-
-**Request query parameters**
+_**Request query parameters**_
```

Comment:
To avoid confusion with ###-sections, which have a similar rendering. Adding italics makes more different.

Reply:
NTC